docs/src/main/sphinx/connector/object-storage-file-formats.rst

```diff
@@ -89,6 +89,13 @@ with Parquet files performed by supported object storage connectors:
         bloom filters by default. The equivalent catalog session property is
         ``parquet_use_bloom_filter``.
       - ``true``
+    * - ``parquet.use-column-index``
+      - Skip reading Parquet pages by using Parquet column indices. The
+        equivalent catalog session property is ``parquet_use_column_index``.
```

Comment:
Iceberg connector doesn't support this session property.

Reply:
Why not? Arent these session properties of the Parquet reader?

Reply:
The old docs indicated that all the same properties are supported by Hive, Iceberg, Hudi and Delta Lake .. is that not correct?

Reply:
I'm not sure about the existing properties. I would recommend searching the code. At least, `parquet_use_column_index` session property is implemented only in Delta and Hive connectors. Iceberg and Hudi don't support the session property.

Reply:
Sigh .. this sucks.. so I can either hedge in the docs or implement the code..

Reply:
It's been discussed before, column index support for iceberg is blocked on the problem mentioned in https://github.com/trinodb/trino/pull/13584#issuecomment-1278756350 and it's not a particularly useful feature.

Reply:
Updated accordingly